tests/TestCase/Mailer/MailerTest.php

```diff
@@ -107,13 +108,16 @@ public function testTransportInvalid(): void
     /**
      * testMessage function
```

Comment:
```suggestion
     * testMessage function
     *
     * @deprecated
```

The deprecated annotations makes it easier to find tests that need to be clean up for the next major.

Reply:
well its also wrapped in `$this->deprecated()` so it seems like duplicate info